cpp/pybind/t/geometry/trianglemesh.cpp

```diff
@@ -962,10 +962,23 @@ or has a negative value, it is ignored.
         box = o3d.t.geometry.TriangleMesh.create_box()
         top_face = box.select_by_index([2, 3, 6, 7])
 )");
+    triangle_mesh.def("remove_non_manifold_edges",
+                      &TriangleMesh::RemoveNonManifoldEdges,
+                      "Remove non-manifold edges from the mesh.");
+
+    triangle_mesh.def("get_non_manifold_edges",
+                      &TriangleMesh::GetNonManifoldEdges,
+                      "allow_boundary_edges"_a = true,
+                      "Return the list consisting of non-manifold edges.");
 
     triangle_mesh.def("remove_unreferenced_vertices",
                       &TriangleMesh::RemoveUnreferencedVertices,
                       "Removes unreferenced vertices from the mesh in-place.");
+
+    triangle_mesh.def("compute_triangle_areas",
+                      &TriangleMesh::ComputeTriangleAreas,
+                      "Compute triangle areas and save it as \"areas\" "
```

Comment:
Just a suggestion. An example cold show how to access the new attribute and sum it to get the total surface area.